Project-Euler/Problem015.js

```diff
@@ -0,0 +1,13 @@
+// https://projecteuler.net/problem=15
+/* Starting in the top left corner of a 2×2 grid, and only being able to move to
+the right and down, there are exactly 6 routes to the bottom right corner.
+How many such routes are there through a 20×20 grid?
+*/
+
+const latticePath = n => {
+
+    for (var i = 1, c = 1; i <= n; i++)
```

Comment:
Please add an explanation for your code as comments. The repository is focused on educating people, so it is important that everyone can understand all the code. Refer to #630 for comment format.